plugins/modules/cloud/misc/proxmox_template.py

```diff
@@ -48,7 +48,8 @@
   template:
     description:
       - the template name
-      - required only for states C(absent), C(info)
+      - required for state C(absent) to delete a template
+      - required for state C(present) to download an appliance container template (pveam)
```

Comment:
```suggestion
      - Required for state C(absent) to delete a template.
      - Required for state C(present) to download an appliance container template (pveam).
```
(Let's start doing at least the modified ones correctly :) )